#188 PHPMD static analysis

refatK opened the pull request:
#188 

Here are the command lines used if curious:

- `phpmd App/ text cleancode,codesize,controversial,design,naming,unusedcode --reportfile report.txt --exclude securimage,Tests
`

- `phpmd App/ html cleancode,codesize,controversial,design,naming,unusedcode --reportfile report.html --exclude securimage,Tests
`

And if you would like to try it yourself simply reinstall dependencies with composer.

refatK committed:
#188 added phpmd to project using composer

refatK committed:
#188 ran static analysis on project, with .txt and .html output

MattYu reviewed (commented):
Looks good, thanks! We might need to look at resetting the config file to exclude test directories

refatK committed:
#188 reports without Test

refatK commented:
@MattYu 
Done

refatK committed:
#188 Merge branches '#188-phpmd' and 'master' of https://github.com/JoComp27/SOEN341-SA1 into #188-phpmd

# Conflicts:
#	composer.json
#	composer.lock

#188 removed composer.lock and made ignored
